src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/AttachCaseCallbackService.java

```diff
@@ -447,14 +447,17 @@ private void updateSupplementaryEvidence(
             callBackEvent.exceptionRecordId
         );
 
+        CaseDetails targetCase = ccdApi.getCase(targetCaseCcdRef, callBackEvent.exceptionRecordJurisdiction);
+
         ServiceConfigItem serviceConfigItem = getServiceConfig(exceptionRecordDetails);
         ProcessResult processResult = ccdCaseUpdater.updateCase(
             callBackEvent.exceptionRecord,
             serviceConfigItem,
             ignoreWarnings,
             callBackEvent.idamToken,
             callBackEvent.userId,
-            targetCaseCcdRef
+            targetCaseCcdRef,
+            targetCase.getCaseTypeId()
```

Comment:
isn't this available from `callBackEvent` (same as `targetCaseCcdRef`)?

Reply:
`targetCaseCcdRef` is for the "search case reference type"  which can be either CCD case reference or external case reference